### PR TITLE
Added IRIS support

### LINKED ISSUE #165: Add support for IrisNet (IRIS) addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

We will likely be reviewing PRs and paying out bounties more or less on a weekly basis. No need to ping us to check your PR, we will definitely get to it.

Thanks for your help!

### PR DESCRIPTION
**Pull request by iRhonin**
## Pending
https://github.com/satoshilabs/slips/pull/995

## Issue number

Closes #165

## List of features added/changed

- IRIS encoder/decoder based on bech32

## Reference
[IRIS is built on top of cosmos-sdk](https://www.irisnet.org/docs/)

## How Has This Been Tested?
- Using bech32 to generate hex format

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by Arachnid** [comment]
I have updated the Travis build to check that entries in the formats, tests, and README lists are in the correct order. Please check your PR and update it to maintain correct ordering in these lists.

File: README.md [hunk 1]
```diff
@@ -70,5 +71,5 @@ This library currently supports the following cryptocurrencies and address forma
  - XTZ (base58check)
  - ZEC (transparent addresses: base58check P2PKH and P2SH, and Sapling shielded payment addresses: bech32; doesn't support Sprout shielded payment addresses)
  - ZIL (bech32)
-
+
```
Comment (makoto) [inline README.md:74]: Please remove extra whitespace

**Review by makoto — CHANGES_REQUESTED** [review]

**Comment by makoto** [comment]
@iRhonin Please resolve conflicts

**Comment by iRhonin** [comment]
@makoto done.

**Comment by makoto** [comment]
Sorry but I noticed that this one came first https://github.com/ensdomains/address-encoder/pull/175

**Pull request closed without merging**